.github/workflows/main.yml

```diff
@@ -16,14 +16,14 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: "1.11.4"
-              otp: "21.3.8"
+              elixir: 1.12.3
```

Comment:
Still keep 1.12 as it just the security support just only ended recently, see https://endoflife.date/elixir